apps/etterlatte-behandling/src/main/kotlin/behandling/revurdering/RevurderingService.kt

```diff
@@ -286,6 +286,11 @@ class RevurderingServiceImpl(
         fritekstAarsak = fritekstAarsak
     ).let { opprettBehandling ->
         behandlingDao.opprettBehandling(opprettBehandling)
+
+        if (fritekstAarsak != null) {
```

Comment:
nitpick: Ser det er brukt `.let` under for akkurat det samme ish

Reply:
Usikker på om jeg synes `?.let` er mer lettlest enn `if(` men la gå 😛